marlowe-dashboard-client/src/Pickup/State.purs

```diff
@@ -0,0 +1,37 @@
+module Pickup.State
+  ( initialState
+  , handleAction
+  ) where
+
+import Prelude
+import Data.Lens (assign)
+import Data.Maybe (Maybe(..))
+import Effect.Aff.Class (class MonadAff)
+import Halogen (HalogenM)
+import MainFrame.Lenses (_card, _pickupState, _screen)
+import MainFrame.Types (ChildSlots, Msg)
+import MainFrame.Types (Action, State) as MainFrame
+import Pickup.Types (Action(..), Screen(..), State)
+
+initialState :: State
+initialState =
+  { screen: GenerateWalletScreen
+  , card: Nothing
+  }
+
+-- Some actions are handled in `MainFrame.State` because they involve
+-- modifications of that state. See Note [State].
+handleAction :: forall m. MonadAff m => Action -> HalogenM MainFrame.State MainFrame.Action ChildSlots Msg m Unit
+handleAction (SetNewWalletNickname nickname) = pure unit -- handled in `MainFrame.State`
```

Comment:
I've never done this before but I wonder if it would make sense to make a function alias `handledInParent = pure unit` ?

Maybe it's stupid, I just don't like the look of it now with these comments

Reply:
🤔 `handleInParent` suggests it would do something. Whereas `pure unit` makes it clear it doesn't do anything (and the comment tells you why).

I've just had a thought while writing this. The way the code is written, the function will never be called with these arguments (the parent handler intercepts the actions first). So maybe the ones that do get called should go first, and then we just have one `handleAction _ = pure unit` at the end (with a comment) for everything else.